_health-care/_js/components/ProgressButton.jsx

```diff
@@ -32,7 +37,9 @@ class ProgressButton extends React.Component {
 ProgressButton.propTypes = {
   onButtonClick: React.PropTypes.func.isRequired,
   buttonText: React.PropTypes.string.isRequired,
-  buttonClass: React.PropTypes.string.isRequired
+  buttonClass: React.PropTypes.string.isRequired,
+  beforeText: React.PropTypes.string,
```

Comment:
Can we add really simple Prop Type tests just making sure the values passed to `beforeText` and `afterText` are strings?